`test_getnode` fails on Darwin

While building for Darwi (MacOS) on nixpkgs:
> _________________________________ test_getnode _________________________________
>
>     @pytest.mark.xfail(sys.platform == "linux", reason="Might fail in Github Actions")
>     def test_getnode() -> None:
> >       assert uuid_utils.getnode() == getnode()
> E       assert 59386474338210 == 266075647966828
> E        +  where 59386474338210 = <built-in function getnode>()
> E        +    where <built-in function getnode> = uuid_utils.getnode
> E        +  and   266075647966828 = getnode()
>
> tests/test_uuid.py:211: AssertionError
> =========================== short test summary info ============================
> FAILED tests/test_uuid.py::test_getnode - assert 59386474338210 == 266075647966828

--- a/tests/test_uuid.py
+++ b/tests/test_uuid.py
@@ -218,8 +218,15 @@ def test_is_safe() -> None:
     assert uuid_utils.uuid4().is_safe is SafeUUID.unknown
 
 
-def test_getnode() -> None:
+@pytest.mark.xfail(reason="getnode source may differ from stdlib in different systems")
+def test_getnode_matches_stdlib() -> None:
     assert uuid_utils.getnode() == getnode()
+
+
+def test_getnode() -> None:
+    node = uuid_utils.getnode()
+    assert 0 < node < (1 << 48)
+    assert uuid_utils.getnode() == node
 
 
 @pytest.mark.skipif(

